etc/config-template.toml

```diff
@@ -785,6 +785,15 @@
 # enable-statistics = true
 # stats-dump-period = "10m"
 
+## RocksDB WAL directory.
+## This config specifies the absolute directory path for WAL.
+## If it is not set, the log files will be in the same directory as data. When you set the path to
+## RocksDB directory in memory like in `/dev/shm`, you may want to set`wal-dir` to a directory on a
+## persistent storage. See https://github.com/facebook/rocksdb/wiki/How-to-persist-in-memory-RocksDB-database .
+## If there are two disks on the machine, storing RocksDB data and WAL logs on different disks can
+## improve performance.
+# wal-dir = ""
```

Comment:
Mention the two WAL can't be put together.

Reply:
In fact, you should also add a check similar to https://github.com/tikv/tikv/blob/f81592b0f18b76e79b4f146124c0c1625867e222/src/config.rs#L2371